packages/rum-core/src/performance-monitoring/performance-monitoring.js

```diff
@@ -186,16 +186,24 @@ class PerformanceMonitoring {
   }
 
   filterTransaction(tr) {
-    var performanceMonitoring = this
-    var transactionDurationThreshold = this._configService.get(
+    const transactionDurationThreshold = this._configService.get(
       'transactionDurationThreshold'
     )
-    var duration = tr.duration()
-    if (
-      !duration ||
-      duration > transactionDurationThreshold ||
-      !tr.spans.length
-    ) {
+    const duration = tr.duration()
+    if (!duration || duration > transactionDurationThreshold) {
+      this._logginService.debug(
+        `Transaction was discarded! transaction duration: ${duration} contains falsy value or it is greater than transactionDurationThreshold: ${transactionDurationThreshold} config`
```

Comment:
What are the conditions under which a transaction's duration is falsy? I think there are two:

 - duration is zero
 - transaction either wasn't started or wasn't ended (or neither; also, not sure if it's actually possible to obtain a transaction without starting it?)

Would it be easy enough to restate the message in those terms? It saves the user understanding the implementation details.